docs/release.md

```diff
@@ -20,313 +23,242 @@ release, those are:
 These artifacts are provided in their respective channels and their natural way
 of distribution.
 
-# Cadence and Timeline
 
-# Minor releases
+# Platform Support
 
-KubeVirt aligns to the [Kubernetes release cycle](https://www.kubernetes.dev/resources/release/).
-But because KubeVirt builds upon Kubernetes, it will trail Kubernetes' release schedule by roughly 2 months, in order to have time to consume and stabilize on top of new releases.
+Every KubeVirt release is
 
-Thus new release branches are cut 3 times a year, or about every 15 weeks.
+ i. targeted towards and life-time coupled with the most recent Kubernetes release
+ii. throughout it's life-time the release is validated against the Kubernetes release
+    in the [support period](https://kubernetes.io/releases/patch-releases/#support-period)
+    at the time of GA of the KubeVirt release
 
-## Guiding principles
+For example:
+* KubeVirt v1.0 is targeted towards Kubernetes v1.27.
+* KubeVirt v1.0 will be supported until Kubernetes v1.27 runs out of support.
+* KubeVirt v1.0 will be validated against all Kubernetes versions being in
+  the support period at the time of the v1.0 GA
 
-The release schedule is build around a few guiding principles:
+Leading to:
+* At v1.27 GA: v1.0 with v1.27, v1.26, v1.25
+* At v1.28 GA: v1.0 with v1.27, v1.26
```

Comment:
I wonder if we want to also drop the testing for these EOL versions or do the best effort.